feat: Add BigQuery source and tool (#463)
googleapis/genai-toolbox: docs/en/resources/tools/bigquery-sql.md

      Example:
      {{
          "id": 123,
          "name": "Alice",
      }}
    parameters:
      - name: id
        type: integer
        description: User ID
      - name: email
        type: string
        description: Email address of the user
```

## Reference

| **field**   |                  **type**                  | **required** | **description**                                                                                  |
|-------------|:------------------------------------------:|:------------:|--------------------------------------------------------------------------------------------------|
| kind        |                   string                   |     true     | Must be "bigquery-sql".                                                                          |
| source      |                   string                   |     true     | Name of the source the SQL should execute on.                                                    |
| description |                   string                   |     true     | Description of the tool that is passed to the LLM.                                               |

> | statement   |                   string                   |     true     | SQL statement to execute on.                                                                     |

Reviewer: ```suggestion
| statement   |                   string                   |     true     | The GoogleSQL statement to execute.                                                                     |
```
